packages/components/molecules/f-restaurant-card/src/components/subcomponents/RestaurantRating/RestaurantRating.vue

```diff
@@ -0,0 +1,168 @@
+<template>
+    <p :class="[$style['c-restaurantCard-rating']]">
+        <!-- Icons -->
+        <star-empty-icon
+            v-if="noRatingsAvailable"
+            aria-hidden="true"
+            data-test-id="ratings-star-empty"
+            :class="[$style['c-restaurantCard-ratingStar']]" />
```

Comment:
As this is a child of `c-restaurantCard-rating`, the naming would make more sense as `c-restaurantCard-rating-star` to show the hierarchy between the two classes.

```suggestion
            :class="[$style['c-restaurantCard-rating-star']]" />
```